jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/applications/ServerlessExplorerNodes.kt

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.applications
+
+import com.intellij.openapi.project.Project
+import icons.AwsIcons
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient
+import software.amazon.awssdk.services.lambda.LambdaClient
+import software.aws.toolkits.jetbrains.core.AwsClientManager
+import software.aws.toolkits.jetbrains.core.explorer.AwsExplorerNode
+import software.aws.toolkits.jetbrains.core.explorer.AwsExplorerPageableNode
+import software.aws.toolkits.jetbrains.core.explorer.AwsExplorerResourceNode
+import software.aws.toolkits.jetbrains.core.explorer.AwsTruncatedResultNode
+import software.aws.toolkits.resources.message
+
+class ServerlessApplicationsNode(project: Project) : AwsExplorerPageableNode<String>(project, message("lambda.applications"), null) {
+
+    private val client: CloudFormationClient = AwsClientManager.getInstance(project).getClient()
+
+    override fun loadResources(paginationToken: String?): Collection<AwsExplorerNode<*>> {
+        val response = client.describeStacks { request ->
+            paginationToken?.let { request.nextToken(it) }
+        }
+
+        val nodes = response.stacks().filterNotNull().filter { stack ->
+            client.describeStackResources { it.stackName(stack.stackName()) }.stackResources().any { it.resourceType() == "AWS::Lambda::Function" }
```

Comment:
Consider putting `AWS::Lambda::Function` into a constant somewhere that can be used throughout the extension

Reply:
Currently its defined in `software.aws.toolkits.jetbrains.services.cloudformation.ResourcesKt#LAMBDA_FUNCTION_TYPE`

Reply:
👍 this was definitely just me being lazy